okhttp/src/main/java/okhttp3/OkHttpClient.kt

```diff
@@ -0,0 +1,829 @@
+/*
+ * Copyright (C) 2012 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3
+
+import okhttp3.Protocol.HTTP_1_1
+import okhttp3.Protocol.HTTP_2
+import okhttp3.internal.Internal
+import okhttp3.internal.Util
+import okhttp3.internal.Util.checkDuration
+import okhttp3.internal.cache.InternalCache
+import okhttp3.internal.connection.Exchange
+import okhttp3.internal.platform.Platform
+import okhttp3.internal.proxy.NullProxySelector
+import okhttp3.internal.tls.CertificateChainCleaner
+import okhttp3.internal.tls.OkHostnameVerifier
+import okhttp3.internal.ws.RealWebSocket
+import okio.Sink
+import okio.Source
+import org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement
+import java.net.Proxy
+import java.net.ProxySelector
+import java.net.Socket
+import java.security.GeneralSecurityException
+import java.time.Duration
+import java.util.ArrayList
```

Comment:
`ArrayList` has alias in Kotlin, so import is not needed

Reply:
Good call! fa4cd4f